Yubico.YubiKey/src/Yubico/YubiKey/Cryptography/AesUtilities.cs

```diff
@@ -152,14 +151,14 @@ public static byte[] AesCbcEncrypt(byte[] key, byte[] iv, ReadOnlySpan<byte> pla
         /// <param name="iv">16-byte initialization vector (IV)</param>
         /// <param name="ciphertext">Input blocks; must be a non-zero multiple of 16 bytes long</param>
         /// <returns>Plaintext of the same length as the ciphertext</returns>
-        public static byte[] AesCbcDecrypt(byte[] key, byte[] iv, ReadOnlySpan<byte> ciphertext)
+        public static ReadOnlyMemory<byte> AesCbcDecrypt(ReadOnlySpan<byte> key, ReadOnlySpan<byte> iv, ReadOnlySpan<byte> ciphertext)
         {
-            if (key is null)
+            if (key == null)
             {
                 throw new ArgumentNullException(nameof(key));
             }
 
-            if (iv is null)
+            if (iv == null)
```

Comment:
In the previous method you use `IsEmpty` property of ReadOnlySpan - but here you nullcheck. What is the difference?